### PR TITLE
[quest] ADDED: 'Shield Mastery' Shaman Fake Rune Quests To QuestDB

### LINKED ISSUE #5605: [quest] 'Shield Mastery' Shaman Rune Quest Missing From QuestDB
## Description

The SoD Shaman rune quest 'Shield Mastery' is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/guide/season-of-discovery/shaman-class-runes-discovery-location
https://www.wowhead.com/classic/spell=410098/engrave-chest-shield-mastery

## Screenshots

N/a

## Questie & WoW version

v9.2.17
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #5605 
Linked To #5443

## Proposed changes

- ADDED: All 'Shield Mastery' Shaman fake rune quests and they are now in the QuestDB. Both Durotar & Mulgore locations are accessible to users.

### CODE REVIEW

File: Database/Corrections/sodQuestFixes.lua [hunk 1]
```diff
@@ -3312,6 +3312,42 @@ function SeasonOfDiscovery:LoadQuests()
...
+            [questKeys.startedBy] = {nil,{450011}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 5,
+            [questKeys.requiredRaces] = raceIDs.TROLL,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Find a Galvanic Icon, loot it and equip the Icon, follow its instructions to kill 10 enemies with Lightning Bolt Rank 1 to receive the rune."},
+            [questKeys.requiredSpell] = -410098,
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3338]: You might want to combine these two quests, as they are absolutely the same for Orc and Troll. To do this all you need to do is to change `requiredRaces` to:

```lua
[questKeys.requiredRaces] = raceIDs.ORC + raceIDs.TROLL,
```

If you combine them make sure to also remove the leftover ID in `SeasonOfDiscovery.lua`

File: Database/Corrections/sodQuestFixes.lua [hunk 2]
```diff
@@ -3312,6 +3312,42 @@ function SeasonOfDiscovery:LoadQuests()
...
+        [90221] = {
+            [questKeys.name] = "Shield Mastery",
+            [questKeys.startedBy] = {nil,{152608,152618,3642}},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3341]: I think this is a leftover before introducing the "fake objects"?

```suggestion
            [questKeys.startedBy] = {nil,{450012}},
```

**Review by BreakBB — CHANGES_REQUESTED** [review]

File: Database/Corrections/sodQuestFixes.lua (hunk 1 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Fixed in: https://github.com/Questie/Questie/pull/5612/commits/dc1eb840eb72214bd6bff48a352c243960e6e840

File: Database/Corrections/sodQuestFixes.lua [hunk 3]
```diff
@@ -3313,6 +3313,30 @@ function SeasonOfDiscovery:LoadQuests()
...
+            [questKeys.questLevel] = 5,
+            [questKeys.requiredRaces] = raceIDs.ORC + raceIDs.TROLL,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Find a Galvanic Icon, loot it and equip the Icon, follow its instructions to kill 10 enemies with Lightning Bolt Rank 1 to receive the rune."},
+            [questKeys.requiredSpell] = -410098,
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
+        [90218] = {
+            [questKeys.name] = "Shield Mastery",
+            [questKeys.startedBy] = {nil,{450011}},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3329]: I guess some merge failed.

```suggestion
            [questKeys.startedBy] = {nil,{450012}},
```

**Review by BreakBB — APPROVED** [review]

**Pull request merged**